src/pages/stage-page/validate-stage-page.ts

```diff
@@ -43,7 +43,8 @@ export const validateTimetableStage = ({
   if (endEvent && endEventKey) {
     const endEventValidation = eventSchema.validate(
       {
-        eventDate: endEvent.eventDate,
+        startEvent: startEvent.eventDate,
```

Comment:
I think naming these "startDate" and "endDate" (or startEventDate and endEventDate) would be more correct as they only contain the value of the date not the whole event

Reply:
Althought careful that `startEvent` and `endEvent` could be confused with the data standard fields?